docs/basic_training/processes.pt.md

```diff
@@ -0,0 +1,950 @@
+---
+description: Material de treinamento básico do Nextflow
+---
+
+# Processos
+
+No Nextflow, um processo (`process`) é a primitiva de computação básica para executar funções estrangeiras (ou seja, scripts personalizados ou ferramentas).
+
+A definição do processo começa com a palavra-chave `process`, seguida pelo nome do processo e, finalmente, o corpo do processo delimitado por chaves.
+
+O nome do process é comumente escrito em letras maiúsculas por convenção.
+
+Um processo básico, usando apenas o bloco de definição `script`, se parece com o seguinte:
+
+```groovy linenums="1"
+process DIGAOLA {
+
+  script:
+  """
+  echo 'Olá mundo!'
+  """
+}
+```
+
+Em exemplos mais complexos, o corpo do processo pode conter até **cinco** blocos de definição:
+
+1. **Diretivas** são declarações iniciais que definem configurações opcionais
+2. **Input** (Bloco de entrada) define o(s) arquivo(s) de entrada esperado(s) e o canal onde encontrá-los
+3. **Output** (Bloco de saída) define o(s) arquivo(s) de saída esperado(s) e o canal para enviar os dados
+4. **When** é uma declaração de cláusula opcional para permitir processos condicionais
+5. **Script** é uma string que define o comando a ser executado pelo processo
+
+A sintaxe completa do processo é definida da seguinte forma:
+
+!!! info ""
+
+    Clique no ícone :material-plus-circle: no código para ver explicações.
+
+```groovy linenums="1"
+process < nome > {
+
+  [ diretivas ] // (1)!
+
+  input: // (2)!
+  < entradas do processo >
+
+  output: // (3)!
+  < saídas do processo >
+
+  when: // (4)!
+  < condição >
+
+  [script|shell|exec]: // (5)!
+  """
+  < script do usuário a ser executado >
+  """
+}
+```
+
+1. Zero, uma ou mais diretivas de processo
+2. Zero, uma ou mais entradas para o processo
+3. Zero, uma ou mais saídas para o processo
+4. Uma condicional booleana opcional para acionar a execução do processo
+5. O comando a ser executado
+
+## Script
+
+O bloco `script` é uma string que define o comando a ser executado pelo processo.
+
+Um processo pode executar apenas um bloco `script`. Deve ser a última instrução quando o processo contém declarações de entrada e saída.
+
+O bloco `script` pode ser uma string de uma ou várias linhas. A de várias linhas simplifica a escrita de scripts não triviais compostos por vários comandos abrangendo várias linhas. Por exemplo:
```

Comment:
Isn't that redundant? Maybe just "O bloco de várias linha simplifica a execução de uma série de comandos."

Reply:
I agree the writing isn't the best, but we shouldn't stray too far from the original material in English. I have written it down to improve in the original material and once fixed there, we can come back here.